src/page-multi-facility/MultiFacilityPage.tsx

```diff
@@ -16,6 +23,21 @@ const MultiFacilityImpactDashboard = styled.main.attrs({
 })``;
 
 const MultiFacilityPage: React.FC = () => {
+  const [facilities, setFacilities] =
+    useState({ data: [] as Array<Facility>, loading: true });
```

Comment:
I dig types and all, but I feel like I'm doing something wrong in this code given the verbosity of all of these `as Array<Facility>`

Reply:
this one is annoying but correct as far as I can tell. you should probably be either guarding against `null` on L31 or accepting `null` here rather than asserting that `getFacilities()` will always return `Array<Facility>`, since we know that to be incorrect.

To save characters you could also just `export type Facilities = Facility[]` from your facility types file?

Reply:
`export type Facilities = Facility[]` is a neat trick ✅